.gitignore

```diff
@@ -1,2 +1,3 @@
 # ignore build directory
+.vscode/
```

Comment:
@trozensztrauch: this looks unnecessary.

Reply:
Great. I have removed that line.